common/determined_common/api/request.py

```diff
@@ -31,13 +32,16 @@ def set_master_cert_name(name: Optional[str]) -> None:
 # Set the bundle if one is specified by the environment. This is done on import since we can't
 # always count on having an entry point we control (e.g., if someone is importing this code in a
 # notebook).
-set_master_cert_bundle(os.environ.get("DET_MASTER_CERT_FILE"))
+f = os.environ.get("DET_MASTER_CERT_FILE")  # type: Optional[Union[str, bool]]
+if f == "False":
```

Comment:
maybe make this case-insensitive, and maybe have the string value be `noverify` or something?